src/Tribe/Cache.php

```diff
@@ -100,7 +100,9 @@ public function set_transient( $id, $value, $expiration = 0, $expiration_trigger
 	 * @return mixed
 	 */
 	public function get( $id, $expiration_trigger = '', $default = false, $expiration = 0, $args = array() ) {
-		$group = in_array( $id, $this->non_persistent_keys ) ? 'tribe-events-non-persistent' : 'tribe-events';
+		$group = isset( array_flip( $this->non_persistent_keys )[ $id ] ) ?
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: isset() must only contain variables (PHP 5.6 compatibility)

<sub>hash: a6605a776b5692a2a07e59b82326b308</sub>